src/MyEventEmitter.js

```diff
@@ -1,14 +1,74 @@
+/* eslint-disable max-len */
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+    this.events[eventName].push(listener);
+  }
+
+  once(eventName, listener) {
+    const onceListener = (...args) => {
+      this.off(eventName, onceListener);
+      listener(...args);
+    };
+
+    this.on(eventName, onceListener);
+  }
+
+  off(eventName, listener) {
+    if (!this.events[eventName]) {
+      return;
+    }
+
+    this.events[eventName] = this.events[eventName].filter(
+      (registeredListener) => registeredListener !== listener,
+    );
+  }
+
+  emit(eventName, ...args) {
+    if (!this.events[eventName]) {
+      return;
+    }
+
+    this.events[eventName].forEach((listener) => {
+      listener(...args);
+    });
+  }
+
+  prependListener(eventName, listener) {
+    if (!this.events[eventName]) {
```

Comment:
This is a code quality issue. You're repeating the same check and array initialization in multiple methods ('on', 'prependListener', 'once', 'prependOnceListener'). This violates the DRY (Don't Repeat Yourself) principle. Consider creating a private method that ensures the existence of the array for a given event.